<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorizationWindow</class>
 <widget class="QMainWindow" name="AuthorizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="HeadingLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Выберите права входа:</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="DescriptionLabel">
      <property name="text">
       <string>Обязательно нажмите одну из кнопок перед вводом логина и пароля!</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="ClientPushButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Клиент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AdminPushButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Админ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="AuthorizationHeadingLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="UsernameLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UsernameLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="PasswordLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="BackPushButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EnterPushButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
